index.html

```diff
@@ -1,6 +1,7 @@
 <html>
     <head>
         <title>World greeter</title>
+	<script src="index.js"></script>
```

Comment:
Maybe add the `defer` keyword here.  I got an error when I ran the app that said "title is not defined".